pkg/controller/seed-controller-manager/mla/mla.go

```diff
@@ -37,7 +37,7 @@ import (
 )
 
 type cleaner interface {
-	cleanUp(context.Context) error
+	CleanUp(context.Context) error
```

Comment:
Why this became a public method?

Reply:
Because it was hard for me to see which functions on the various reconcilers are called by other controllers and which are only called internally. It's not super duper important, but I liked to see `func (r *reconciler) CleanUp()` and immediately know "oh, this is probably called from somewhere outside". "CleanUp()" was similar to "Reconcile()" to me.